lib/kitchen/figure_element.rb

```diff
@@ -47,16 +47,42 @@ def subfigure?
       parent.name == 'figure'
     end
 
+    # Returns true if the figure is unnumbered
+    #
+    # @return [Boolean]
+    #
+
+    def unnumbered_figure?
+      has_class?('unnumbered')
+    end
+
     # Returns true unless the figure is a subfigure or has the 'unnumbered' class,
     # unless the figure has both the 'unnumbered' and the 'splash' classes.
     #
     # @return [Boolean]
-    #
+
+    # TODO, remove this method from other books recipes to prevent counting unnumbered baked figures
     def figure_to_bake?
-      return false if subfigure? || (has_class?('unnumbered') && !has_class?('splash') && !caption)
+      return false if subfigure? || (has_class?('unnumbered') &&
+                                    !has_class?('splash') && !caption &&
+                                    !title)
```

Comment:
I have updated `figure_to_bake` method to contain a title and left the method in place to avoid breaking the other recipes. Below I have created two separate methods for baking numbered and unnumbered figures (`figure_to_bake_without_count`, `figure_to_bake_and_count` ). The reason for it was that when I tried to use `figure_to_bake` in WH recipe I couldn't exclude unnumbered figures from counting while looping through all figures that should be baked: 
![image](https://user-images.githubusercontent.com/55203598/136270688-bc8a7705-68ec-417c-892f-55acc26d50e3.png)

also attempts to define the figures to be numbered under condition gave no results: 

![image](https://user-images.githubusercontent.com/55203598/136270835-605c5777-0e00-4592-896d-7270e5e677dc.png)
![image](https://user-images.githubusercontent.com/55203598/136270991-f39a0a64-f686-4107-b7d2-580a438de6e9.png)
In various books amount of figures can differ and in some of the them using `figure_to_bake` could not break the numbering (f.ex. if there will be no unnumbered figure in a chapter).

My second thought was to create two separate directions for baking unnumbered and numbered figures. If you think it will be a better idea or you will have some other suggestions please let me know :relaxed: